pytest.ini

```
@@ -1,7 +1,7 @@
 [pytest]
-addopts = -v -ra --cov
-python_paths = src/
+addopts = --cov -v -ra --runslow
```

We encourage the use of long flag names when available and might as well alphabetize the flags while we're at it:
```suggestion
addopts = --cov -ra --runslow --verbose
```

---

good to know. resolved with [d8d9366](https://github.com/cisagov/domain-manager-api/pull/553/commits/d8d93668304b7b3768f38f30d39248e149fff96f)